working-groups/wg-talent-plan-courses.md

```diff
@@ -0,0 +1,29 @@
+# Talent Plan Courses Working Group
+
+## Goals
+
+THe Talent Plan Courses Working Group aims to develop a series of courses to help more peole enter the field of computer infrastructure system and the TiDB ecosystem.
```

Comment:
```suggestion
The Talent Plan Course Working Group aims to develop a series of courses to help people enter the field of computer infrastructure system and TiDB ecosystem.
```

Reply:
This goal is not concert. Please describe more about:
1. what courses are to be developed.
2. the RFC for talent plan courses.